cli/args/flags.rs

```diff
@@ -283,7 +282,7 @@ impl RunFlags {
 pub struct ServeFlags {
   pub script: String,
   pub watch: Option<WatchFlagsWithPaths>,
-  pub port: NonZeroU16,
+  pub port: u16,
```

Comment:
Please update the flag help output to tell that passing 0 will pick a random free port.

Reply:
Updated. Thanks for disabling the auto-merge.